docs/design/multi-scheduler.md

```diff
@@ -0,0 +1,213 @@
+````
+- resourceGroup: XXX                   
+  object:
+    key: annotation                             # set the field and the value to be matched
+    value:
+    - "volcano.sh/resource-group: XXX"          # XXX is the value in resourceGroup field
+````
+
+For example
+````
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: volcano-admission-configmap
+  namespace: volcano-system
+data:
+  volcano-admission.conf: |
+    resourceGroups:
+    - resourceGroup: management                    # set the resource group name
+      object:
+        key: namespace                             # set the field and the value to be matched
+        value:
+        - mng-ns-1
+      schedulerName: default-scheduler             # set the scheduler for patching
+      tolerations:                                 # set the tolerations for patching
+      - effect: NoSchedule
+        key: taint
+        operator: Exists
+      labels:
+        volcano.sh/nodetype: management           # set the nodeSelector for patching
+    - resourceGroup: cpu
+      object:
+        key: annotation
+        value:
+        - "volcano.sh/resource-group: cpu"
+      schedulerName: volcano
+      labels:
+        volcano.sh/nodetype: cpu
+    - resourceGroup: gpu                          # if the object is unsetted, default is:  the key is annotation,
+      schedulerName: volcano                      # the annotation key is fixed and is "volcano.sh/resource-group", The corresponding value is the resourceGroup field
+      labels:
+        volcano.sh/nodetype: gpu
+````
+
+### The pod mutate process 
+![](./images/pod-webhook-mutate.png)
+
+If the pod matches serval resource groups, volcano will match the resource group by sort.
+
+
+## Usage
+### case 1 
+Default scheduler for system daemon, and volcano scheduler for biz workload
+Here is a cluster as the following:
+
+|node| label | taint|
+|----|-----|-----|
+|node1| mng-1-label: value-1| management-taint:NoSchedule|
+|node2| none| none|
+
+|pod | kind | namespace|
+|----|----|----|
+|deployment-A|deployment| mng-ns-1| 
+|volcano-job-B|volcano job | default|
+
+1. Edit volcano-admission-configmap 
+````
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: volcano-admission-configmap
+  namespace: volcano-system
+data:
+  volcano-admission.conf: |
+    resourceGroups:
+    - resourceGroup: management                    # set the resource group name
+      object:
+        key: namespace                             # set the field and the value to be matched
+        value:
+        - mng-ns-1
+      schedulerName: default-scheduler             # set the scheduler for patching
+      tolerations:                                 # set the tolerations for patching
+      - effect: NoSchedule
+        key: management-taint
+        operator: Exists
+      labels:
+        volcano.sh/nodetype: management           # set the nodeSelector for patching

```

Comment:
this label is wrong

Reply:
> this label is wrong

what's wrong about the label?

Reply:
in this case, it should be `mng-1-label: value-1`?

Reply:
Got it . I modify the node info from "mng-1-label: value-1" to "volcano.sh/nodetype: management"

Reply:
done